android/app/src/main/java/ru/spbstu/amd/learnbraille/serial/UsbSerial.kt

```diff
@@ -0,0 +1,117 @@
+package ru.spbstu.amd.learnbraille.serial
+
+import android.app.PendingIntent
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.hardware.usb.UsbDevice
+import android.hardware.usb.UsbDeviceConnection
+import android.hardware.usb.UsbManager
+import com.felhr.usbserial.UsbSerialDevice
+import com.felhr.usbserial.UsbSerialInterface
+import ru.spbstu.amd.learnbraille.database.BrailleDots
+import timber.log.Timber
+
+class UsbSerial// TODO make this class a singleton and initialize on startup; allow multiple connection attempts
+    (usbManager: UsbManager, context: Context) {
```

Comment:
Ctor args should be at the previous line

Reply:
done